dev-docs/get-started-with-prebid-server.md

```diff
@@ -43,6 +43,8 @@ Prebid Server is an open source project.  [The source code is hosted under the P
 ## Step 2. Download Prebid.js with Prebid Server enabled
 
 - Go to [the Prebid.org download page]({{site.baseurl}}/download.html), select all the demand adapters you want to work with, and include "Prebid Server".
+  - Some Prebid Server demand adapters may not have a corresponding client-side adapter that's present on the download page.  In this case, just ensure to select "Prebid Server" as part of your build and you will be able to interact with these Prebid Server only bidders.
+  - In lieu of the point above, we still strongly recommend to select any listed demand adapters in your build.  These additional selections will allow any client-side features (such as userSyncs) to function as well as allow you to easily use any of the adapter's currently registered aliases.
```

Comment:
'In lieu of' means 'Instead of' -- I think you mean "despite the point above", but I recommend dropping the whole clause.